recipes/spingo/meta.yaml

```diff
@@ -0,0 +1,25 @@
+about:
+  home: https://github.com/GuyAllard/SPINGO
+  license: ''
+  summary: 'SPecies level IdentificatioN of metaGenOmic amplicons'
+build:
+  number: 0
+  string: py{{CONDA_PY}}_boost{{CONDA_BOOST}}
+package:
+  name: spingo
+  version: 'v1.3'
+requirements:
+  build:
+      - gcc
+      - libgcc >=3.4.15
+      - boost {{CONDA_BOOST}}*
+  run:
+      - libgcc >=3.4.15
+      - boost {{CONDA_BOOST}}*
+source:
+  git_rev: v1.3
+  git_url: https://github.com/GuyAllard/SPINGO.git
```

Comment:
Can you please user here a tarball and a checksum. With this we can backup the tarballs better.